### ISSUE TITLE
How can I remove an activator template?

### ISSUE THREAD

**Issue by aluncoppack** [typesafehub/activator#1144]
I'd like to remove the outdated, non working https://www.lightbend.com/activator/template/mdtp-frontend-microservice

**Comment by eed3si9n** [comment]
I assume you work for HMRC?

**Comment by aluncoppack** [comment]
Yep. See here https://github.com/orgs/hmrc/people?utf8=%E2%9C%93&query=alun

**Comment by eed3si9n** [comment]
Not seeing you there. Is it public?

**Comment by aluncoppack** [comment]
It is now :)

**Comment by corruptmemory** [comment]
I marked the template to not be indexed.  It may take a little while for it to disappear, but it will. :-)

**Comment by aluncoppack** [comment]
many thanks
